pkg/controller/disruption/disruption.go

```diff
@@ -420,7 +420,8 @@ func (dc *DisruptionController) getPdbForPod(pod *v1.Pod) *policy.PodDisruptionB
 	}
 	return pdbs[0]
 }
-
+// this function return pods using the PodDisruptionBudget object
+// IMPORTANT NOTE : the returned pods should not be modified (i.e used as read-only objects)
```

Comment:
s/not/NOT/

Also remove the "(i.e used as read-only objects)"

And make comments sentences - start with capital letter and end with dot.